build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -134,14 +146,62 @@ public IEnumerable<string> ListBlobs(string virtualDirectory)
                 }
             }
 
+            ResetLeaseRenewalTaskState();
```

Comment:
I missed throw directly underneath this, expected it was returning something. Makes sense